code-of-conduct.md

```diff
@@ -39,7 +39,7 @@ Code for America is dedicated to providing a harassment-free experience for ever
 Harassment includes but is not limited to: offensive verbal or written comments related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, unwelcome sexual attention, unwarranted exclusion, gaslighting, tone-policing, sealioning, and patronizing language or action.
 If a participant engages in harassing behavior, the organizers may take any action they deem appropriate, including warning the offender or expulsion from Code for America network activities, events, and digital forums.
 
-Harassment can also include persistent private communications (direct messages, text messages, etc). CDA members should publicly get permission from anyone with whom they would like to communicate privately before initiating such communication. It is up to the initiator of the conversation to gain this permission and, when in doubt, they should reaffirm they have this permission publicly. This permission can be rescinded at any time and, upon being told that private communications are no longer desired, the communication must cease immediately without further comment - including any messages intended to affirm that the request to cease communications is understood. It is expected that once permission to communicate privately is rescinded, no other words will be exchanged privately without regaining permission in a public forum. Failure to obtain permission or failure to cease communications when permission has been revoked constitutes harassment and is not tolerated under the CDA Code of Conduct.
+Harassment can also include persistent private communications (direct messages, text messages, etc). CDA members should publicly get permission from anyone with whom they would like to communicate privately before initiating such communication. It is up to the initiator of the conversation to gain this permission and, when in doubt, they should re-confirm they have this permission, publicly. This permission can be rescinded at any time and, upon being told that private communications are no longer desired, the communication must cease immediately without further comment - including any messages intended to confirm that the request to cease communications is understood. It is expected that once permission to communicate privately is rescinded, no other words will be exchanged privately without regaining permission in a public forum. Failure to obtain permission or failure to cease communications when permission has been revoked constitutes harassment and is not tolerated under the CDA Code of Conduct. 
```

Comment:
I would like to remove the requirement to get public permission from someone before initiating a private conversation with them. CDA members will have committed to the CoC, which defines harassment and other unacceptable behavior, and I think it's pretty standard for members of a community to assume they can have one-on-one conversations without a public ask-answer process taking place.
I think the definition of unacceptable behavior will protect from bad actions in initial communications and the right to request no further communications will protect from subsequent issues.

Reply:
No one should assume they have the "right" to contact anyone privately. Just because you belong to a group gives you no right to contact anyone privately without their express consent.

Private messages are an intimate form of communication and consent must be expressly given, not assumed, not taken. It has been shown repeatedly to not be a necessary caveat to have for protection especially for members of marginalized groups in the community.

Reply:
I don't see a difference between a DM on Slack and a single-recipient email. We aren't saying someone needs to get public sign-off before sending an email, are we?

Could you share examples of the repeated need for pre-approval that would not be able to be resolved by the other parts of the CoC?

Reply:
I'm with Margeaux on this, obviously. We may have to interact in public chat rooms and places while doing what we need to do for CDA but you have absolutely no right to contact me privately (you/me used reflexively here) just like you have no right to pull me aside to a private place at an event to talk to me. These are things you ask permission to do and, when consent is given, you have the conversation in private.

Reply:
It's literally as simple as "Hey, Bret, may I DM you?" If you say yes, I DM you. When you tell me you'd prefer that I not DM you anymore, I silently accept that and don't say another word in the DM. To do anything else is harassment. 

With a single-recipient email sent in the course of your work with CDA, I'd venture to say that you will either a) already have permission as the email address will have been willingly given to you by the intended recipient or b) your first email will receive no reply or a request to cease at which point you silently accept their right to be left alone.

Reply:
I should also address concerns that this is already covered elsewhere in the CoC, I disagree - otherwise we wouldn't have had reports of people DMing others without permission and continuing that conversation after they have been explicitly told to stop.

Reply:
If you’ve been given an individual’s email address by that individual, that would be consent. 

If you’ve gotten the email address in any other manner, that would not be consent, so you’d need to ask.

Reply:
People who are not members of marginalized groups often have a difficult time with consent. 

This requirement allows members to actually be afforded the opportunity to give or withhold consent in regards to their interactions with individual members vs privileged groups assuming what they consider theirs by right.

Joining a group does not give every single member of that group the right to contact me privately. Assuming consent is not conducive to creating a safe space for marginalized groups.

Reply:
I do see sending DMs that way. When you pull me aside physically to have a private discussion (like at an event or a meeting), you are putting me in a place where there are not as many observers / witnesses to what you may say. If your behavior were to get out of hand (again, you/me used reflexively), I would be limited on the number of witnesses I could call to support claims of harassment/abuse/misconduct. In DMs, we're dealing with the same thing. Forcing me in to a DM without consent puts me in the position to where I would be limited in terms of supporting any claims of misconduct. Screenshots and even entire websites can be faked. A DM is no different so the credibility of screenshots would leave room for doubt. So... again... no witnesses. 

Needless to say, if I'm going to let someone put me in a place where my access to backup / witnesses is limited - it's going to be with my consent and only because I trust that they will behave in a way that is appropriate.

As far as overlooking the custom of seeking permission first, the bylaws we're proposing would support a warning before any real disciplinary action would take place. If you DM a member without permission (again, you is reflexive and not indicative of you personally), they can report the behavior. Assuming the first violation of this policy wasn't loaded with some other form of harassment, a simple "Hey, we don't do that here." would be sufficient to attempt to correct the issue. Now, if the first unsolicited DM a member ever sent included something like hate speech, they'd be out (imo).

In regards to the rule requiring a person to cease communications - thank you for your support there.

Reply:
@cheapwebmonkey, if you are implying that I have a difficult time with consent or view consent as a negative, I reject that assertion completely. As I said, I fully support the right for CDA members to control who contacts them privately. I simply disagree with the need to obtain pre-approval to start a conversation with a fellow member a collaborative organization.

I think I have stated my point of view clearly, and I've added my review to this PR.

Reply:
To tag on to what Margeaux said, as a member of a marginalized community myself - I would appreciate the ability to limit my private interactions with people who may not be so supportive of me or whose behavior makes me uncomfortable. This should be proactively offered - not reactively.

Reply:
Agree, Aaron . As a member of a marginalized group, my consent being given versus assumed is paramount.

That others don’t see the internet as having the for a safe and equitable space

Reply:
I’m failing to see how respecting boundaries is negative. And one cannot assume person a has the same boundaries as ones own, so best to ask.